process/peer/process.go

```diff
@@ -423,7 +423,7 @@ func (vs *validatorStatistics) getValidatorDataFromLeaves(
 
 func getActualList(peerAccount state.PeerAccountHandler) string {
 	savedList := peerAccount.GetList()
-	if peerAccount.GetUnStakedEpoch() == 0 {
+	if peerAccount.GetUnStakedEpoch() == math.MaxUint32 {
```

Comment:
set unstaked epoch to maxuint when initializing a new node. This can be done in NewPeerAccount